### ISSUE TITLE
Connection params in  User settings

### ISSUE THREAD

**Issue by mickeypearce** [zabel-xyz/plsql-language#51]
Hi @zabel-xyz ,
```
"plsql-language.connections": [],
"plsql-language.connection.activeInfos": ""
```
are automatically added to User settings and are very persistent. :)  I can't delete them as they immediatelly appear back. Is this intentional?

**Comment by zabel-xyz** [comment]
Hello @mickeypearce,
No, you found a little bug...
If there is no connections settings (a new feature that I'm working on), there is no reason to appear back like that !. I'll fix it...

**Comment by zabel-xyz** [comment]
Fixed in 1.4.1

**Comment by mickeypearce** [comment]
@zabel-xyz, it happens again in the latest version (1.6.0)

**Comment by zabel-xyz** [comment]
I'll see and fix it...

**Comment by zabel-xyz** [comment]
Fix in version 1.6.1
